samples/bot-adaptive-card-actions/csharp/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send adaptivecard actions using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "12/27/2022 12:30:00 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-adaptivecard-actions-csharp
+---
+
+# Get Adaptive Card and Suggested Actions
+
+This sample shows the feature where user can send Adaptive Card actions using bot.
```

Comment:
Add Documentation link that you have referred (Microsoft Docs)

Reply:
fixed